src/resources/plugins/i18n/i18n-target.ts

```diff
@@ -15,7 +15,7 @@ export class I18NTarget {
     this.myDate = new Date();
     this.myDate.setHours(this.myDate.getHours() - 2);
 
-    relativeTime.addTranslationResource("")
+    relativeTime.addTranslationResource("", "");
```

Comment:
Is it better to add a third and custom locale with custom resources here? Otherwise it can be removed altogether. I think I did with something similar in mind and somehow lost with other stuffs :P

Reply:
agree lets get it removed, if we need to test this it should be done more prominently. For now I think we're good